### PR TITLE
Store roles & qualifications for Users

### PR DESCRIPTION
**Pull request by brownbearsoftware**
**Store roles & qualifications for Users**
Add two new tables for storing roles & qualifications

**Implement GET /profile endpoint**
This returns info for the logged in user for the frontend to use

### CODE REVIEW

**Comment by brownbearsoftware** [comment]
[Basic allocation of applications to assessors](https://trello.com/c/3HG6wz6x/793-basic-allocation-of-applications-to-assessors)

File: src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt [hunk 1]
```diff
@@ -22,5 +26,53 @@ data class UserEntity(
...
+  val role: UserRole
+)
+
+enum class UserRole {
+  ASSESSOR,
+  MATCHER,
+  MANAGER,
+  WORKFLOW_MANAGER,
+  APPLICANT,
+  ADMIN
```
Comment (pezholio) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt:57]: Should these enums be lowercase too?

**Review by pezholio — APPROVED** [review]
LGTM - really clear 👍 Dunno if I'm on the right track with those enums - feel free to ignore me!

File: src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt (hunk 1 above)
Comment (pezholio) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt:57]: Ah, that's fine then 👍

**Pull request merged**